app/controllers/users_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
   end
 
   def show
-    @user = User.find(session[:user_id])
+    render file: "/public/404" unless current_user
```

Comment:
Yeah that works. A later user story will address the actual contents of the user profile, and we can get it working when we get there.